tests/Cache/MemcachedCacheTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of SeAT
+ *
+ * Copyright (C) 2015, 2016, 2017  Leon Jacobs
+ * Copyright (C) 2017 HgAlexx
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+use Seat\Eseye\Cache\MemcachedCache;
+use Seat\Eseye\Containers\EsiResponse;
+
+
+class MemcachedCacheTest extends PHPUnit_Framework_TestCase
+{
+    /*
+     * @var MemcachedCache
+     */
+    protected $memcached_cache;
+    
+    protected $esi_response_object;
+
+    public function setUp()
+    {
+        // Memcached: change config for test if needed
+        //$configuration = Seat\Eseye\Configuration::getInstance();
+        //$configuration->memcached_host = '192.168.0.151';
+        //$configuration->memcached_port = 11211;
+        //$configuration->memcached_compressed = true;
+
+        // Set the cache
+        $this->memcached_cache = new MemcachedCache();

```

Comment:
We are going to have to create a mock instance of memcache here (much like the redis one) for the tests, otherwise there will be errors like there currently are in Travis for this PR. Having to manually edit tests to make them run is a pain ;)

Reply:
I wasn't sure how to do that but it was pretty easy, done.